doc/api/HostDB.md

```diff
@@ -288,3 +402,46 @@ Examples
   ]
 }
 ```
+
+#### Hosts
+
+###### Request
+```
+/hostdb/hosts/ed25519:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
```

Comment:
ah, because the caller needs it to construct the request?

It is a little confusing to have three different representations of the pubkey flying around. In the string, it's hex; in JSON, it's base64; and in memory, it's binary.